pkgpanda/build/test_util.py

```diff
@@ -0,0 +1,86 @@
+import pkgpanda.build
+
+
+def test_hash_files_in_folder(tmpdir):
+    hash_files_in_folder = pkgpanda.build.hash_files_in_folder
+
+    with tmpdir.as_cwd():
+        # Empty folder
```

Comment:
Nit cntd.

https://pragprog.com/magazines/2012-01/unit-tests-are-first -> check section "I for isolated"
http://ricardogeek.com/docs/clean_code.pdf -> "Unit tests" chapter if you are have time for a longer read.